src/Services/SeoService.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace IO\Services;
+
+use Plenty\Plugin\Http\Request;
+
+/**
+ * Class SeoService
+ *
+ * @package IO\Services
+ */
+class SeoService
+{
+    public function __construct()
+    {
+    }
+
+    public function getRobotsInformation()
+    {
+        /** @var CategoryService $categoryService */
+        $categoryService = pluginApp(CategoryService::class);
+        $currentCategory = $categoryService->getCurrentCategory();
+        $robots = str_replace('_', ', ', $currentCategory->details[0]->metaRobots);
+
+        if (strlen($currentCategory->details[0]->canonicalLink) !== 0) {
+            return $robots;
+        }
+
+
+        $request = pluginApp(Request::class);
+        $queryParameters = $request->all();
+        unset($queryParameters['plentyMarkets']);
+
+        if (TemplateService::$currentTemplate === 'tpl.search') {
```

Comment:
Hier kannst du inzwischen `pluginApp(ShopUrls::class)->is(RouteConfig::SEARCH)` verwenden.